### PR TITLE
[Docs] Added icon for external links in nav

### PR DESCRIPTION
**Pull request by joecritch**
- **Added an icon** for any external links in the Docs left-hand navigation.
- **Deleted interstitial pages**  ("Complementary Tools" and "Examples"), in favor of linking directly to GitHub Wiki pages. (The icon now signifies exiting the Docs.)

**Screenshot**

screen shot 2016-02-22 at 22 42 33

**Note**

The icon was designed from scratch.

### CODE REVIEW

**Comment by jimfb** [comment]
We may want to keep the markdown pages around in order to avoid breaking links.  @zpao?

**Comment by joecritch** [comment]
@jimfb Fair point! I've updated the commit to keep them in. We can deal with cleanup as a separate concern.

File: docs/css/react.scss [hunk 1]
```diff
@@ -260,6 +260,22 @@ h1, h2, h3, h4, h5, h6 {
     &.active {
       color: $primary;
     }
+
+    &[href*="http"] {
```
Comment (zpao) [inline docs/css/react.scss]: Can we just add a class to these links instead? https://github.com/facebook/react/blob/de09e0acd82d2de955af4c395fd9066292adef70/docs/_plugins/sidebar_item.rb#L6-L7

Comment (joecritch) [inline docs/css/react.scss]: @zpao Sure, Updated. If the plugin sees an href attribute, it'll add an "external" class.

**Comment by zpao** [comment]
Seems good. Where did the icon come from (want to make sure we have appropriate permissions)?

**Comment by joecritch** [comment]
@zpao Cool. It's from scratch. Sent you the PSD.

**Comment by jimfb** [comment]
Ok, I'm cool with this, let's ship it.

**Comment by jimfb** [comment]
Thanks @joecritch

File: docs/_plugins/sidebar_item.rb [hunk 2]
```diff
@@ -4,7 +4,14 @@ def sidebar_item_link(item)
...
       href = item["href"] || "/react/docs/#{itemID}.html"
-      className = pageID == itemID ? ' class="active"' : ''
+      classes = []
+      if pageID == itemID
+        classes.push("active")
+      end
+      if item["href"]
+        classes.push("external")
+      end
+      className = classes.size ? "class=\"#{classes.join(' ')}\"" : "";
```
Comment (zpao) [inline docs/_plugins/sidebar_item.rb]: This isn't right, here's what gets generated for empty items. I have another thing I want to change so going to follow up.

```diff
           <li>
-            <a href="/react/docs/webcomponents.html">Web Components</a>
+            <a href="/react/docs/webcomponents.html"class="">Web Components</a>

           </li>
```

Comment (joecritch) [inline docs/_plugins/sidebar_item.rb]: Hmm, I'm getting a different output locally (ruby 2.2)

Comment (zpao) [inline docs/_plugins/sidebar_item.rb]: `bundle exec`? It shouldn't really be a bundler issue though because 0 isn't falsey in Ruby.

Comment (joecritch) [inline docs/_plugins/sidebar_item.rb]: Ah shoot, didn't realise I needed to re-boot `bundle exec jekyll` after each plugin change. Hence the false positive. Sorry for the mess.

Comment (zpao) [inline docs/_plugins/sidebar_item.rb]: No worries :) #6151 in case you're interested

**Pull request merged**